### PR TITLE
Simple Ingress propogation to member clusters

### PR DESCRIPTION
**Pull request by shashidharatd**
This pr implements simple propogation of Ingress objects to federated clusters and is the first step to implement `Federated Ingress` as described in https://github.com/kubernetes-sigs/federation-v2/issues/96

/cc @kubernetes-sigs/federation-wg

### CODE REVIEW

**Comment by marun** [comment]
@shashidharatd Modulo the single comment, this looks good to me.

File: pkg/apis/core/v1alpha1/zz_generated.kubebuilder.go [hunk 1]
```diff
@@ -303,7 +303,6 @@ var (
 			Scope: "Namespaced",
 			Validation: &v1beta1.CustomResourceValidation{
 				OpenAPIV3Schema: &v1beta1.JSONSchemaProps{
-					Type: "object",
```
[the remark this answers is not in the document]
Comment (marun) [reply pkg/apis/core/v1alpha1/zz_generated.kubebuilder.go]: Reference: https://github.com/kubernetes-sigs/federation-v2/blob/master/scripts/deploy-federation.sh#L121

**Comment by shashidharatd** [comment]
@marun, updated the code for your comment. ptal, Thanks !

**Pull request merged**